File: app/src/main/java/com/sortedqueue/programmercreek/util/ItemTouchHelperAdapter.java
package com.sortedqueue.programmercreek.util;

/**
 * Created by Alok Omkar on 2017-03-09.
 */
public interface ItemTouchHelperAdapter {
    void onItemMove(int fromPosition, int toPosition);
    void onItemDismiss(int position);
}

File: app/src/main/java/com/sortedqueue/programmercreek/util/ItemTouchHelperAdapter.kt
package com.sortedqueue.programmercreek.util

/**
 * Created by Alok Omkar on 2017-03-09.
 */
interface ItemTouchHelperAdapter {
    fun onItemMove(fromPosition: Int, toPosition: Int)
    fun onItemDismiss(position: Int)
}
